source/resiliency.txt

```diff
@@ -12,12 +12,140 @@ Application and Database Resiliency
    :depth: 1
    :class: onecol
 
-Intro statement
+MongoDB |service| is a modern, highly-performant database that is purpose-built 
+to maintain uptime regardless of infrastructure outages, system maintenance and more.
 
 {+service+} Features and Best Practices for Resiliency
 ------------------------------------------------------
 
-Content here
+Database Replication
+~~~~~~~~~~~~~~~~~~~~
+
+|service| {+clusters+} consist of a minimum of three nodes and can be increased 
+to any odd number of nodes you require. Data from your application is first written 
+to a primary node, and that data is then incrementally replicated and stored on 
+all secondary nodes within your cluster.
+
+By default, {+cluster+} nodes are distributed across availability zones within 
+one of your chosen cloud provider's availability regions. For example, if your 
+cluster is deployed to the cloud provider region ``us-east``, |service| deploys 
+nodes to ``us-east-a``, ``us-east-b`` and ``us-east-c`` by default. 
+
+Self-Healing Deployments
+````````````````````````
+
+|service| {+clusters+} must consist of an odd number of nodes, because only one 
+node can be elected as the primary node to and from which your application writes 
+and reads directly. 
+
+In the event that a primary node is unavailable, because of infrastructure 
+outages, maintenance windows or any other reason, |service| {+clusters+} self-heal by 
+converting an existing secondary node into your primary node to maintain 
+database availability. 
+
+Maintenance Window Uptime
+`````````````````````````
+
+|service| maintains uptime during scheduled maintenance by applying updates in 
+a rolling fashion to one node at a time. During this process, |service| elects a new 
+primary when necessary just as it does during any other unplanned primary node 
+outage.  
+
+Monitoring
+~~~~~~~~~~
+
+|service| provides `built-in tools <https://www.mongodb.com/docs/atlas/monitoring-alerts/>`__ to monitor cluster performance, query performance 
+and more. Additionally, |service| integrates easily with `third-party services <https://www.mongodb.com/docs/atlas/tutorial/third-party-service-integrations/#std-label-third-party-integrations>`__. 
```

Comment:
Can we add:

- A sentence on why this matters (if you have visibility into cluster performance issues, you can address issues quickly)
- A link to :ref:`arch-center-monitoring-alerts` to learn more